packages/babel-helper-define-polyfill-provider/src/imports-cache.ts

```diff
@@ -1,8 +1,5 @@
-// @flow
-
 import type { NodePath } from "@babel/traverse";
```

Comment:
We might have to add `@types/babel__traverse` to this package's `devDependencies`